region/brinstar/blue.json

```diff
@@ -874,21 +874,56 @@
           "to": [
             {
               "id": 3,
-              "requires": [
-                {"or": [
-                  "canDamageBoost",
-                  "SpeedBooster",
-                  "HiJump",
-                  "canUseFrozenEnemies",
-                  "canFly",
-                  "canSpringBallJumpMidAir",
-                  "canUnmorphPbBoost"
-                ]}
+              "strats": [
+                {
+                  "name":"Base",
```

Comment:
TAS strat when

Reply:
Yeah I think I'm gonna go ahead and leave this out for now.

This did remind me the strats with enemies need `ZebesAwake` though.